### ISSUE TITLE
[FlexRow] Consider using `columnGap` instead of `spacing` for vertical items' separation by default

### ISSUE THREAD

**Issue by vasilii-kovalev** [epam/UUI#2034]
### Clearer children borders

With `spacing`:
```json
{
  "spacing": "6"
}
```

<img width="300" alt="Image of several components inside FlexRow component when `spacing` property is used" src="https://github.com/epam/UUI/assets/10310491/3abb6432-4e0f-4626-a9c5-9ae387478219">

With `columnGap`:

```json
{
  "spacing": null,
  "columnGap": "6"
}
```

<img width="297" alt="Image of several components inside FlexRow component when `columnGap` property is used" src="https://github.com/epam/UUI/assets/10310491/04891635-feb5-4cc3-a5aa-65576719fa17">

### Great overall support in browsers

Support of `column-gap` property is almost the same as support of the Flexbox itself:

* [Flexbox page on Can I Use](https://caniuse.com/flexbox) (≈ 98%)
* [`column-gap` property page on Can I Use](https://caniuse.com/mdn-css_properties_column-gap_flex_context) (≈ 94%)

## To Do

1. Adjust the [default parameters](https://github.com/epam/UUI/blob/main/loveship/components/layout/FlexItems/FlexRow.tsx#L24-L38) in the `FlexBox` component
2. Adjust mockups and guidelines

**Comment by vik753** [comment]
@vasilii-kovalev 
We decided to deprecate property spacing and recommend using property columnGap instead. We decided this because we know many cases of using negative margins when elements wrap to the next line for example. To avoid breaking changes we will not change the spacing implementation.

**Comment by NatalliaAlieva** [comment]
Released in 5.7.0 ver.
